### PR TITLE
Kiali-561 Update package.json libraries for sprint 4

### PR DESCRIPTION
**Pull request by mtho11**
Bump latest versions of libs for sprint 4

### CODE REVIEW

**Comment by josejulio** [comment]
@mtho11 Looks like you didn't commit your package-lock.json or forgot to do a last `npm install` https://travis-ci.org/kiali/kiali-ui/builds/367224784#L4542

File: package.json [hunk 1]
```diff
@@ -13,15 +13,15 @@
     "cytoscape-klay": "3.1.1",
     "cytoscape-popper": "^1.0.1",
     "lodash": "4.17.5",
-    "patternfly": "3.43.0",
-    "patternfly-react": "1.16.4",
+    "patternfly": "3.45.0",
+    "patternfly-react": "1.16.6",
```
Comment (josejulio) [inline package.json]: +1 
I'll need this one for the [Notification Drawer](https://rawgit.com/patternfly/patternfly-react/gh-pages/index.html?selectedKind=NotificationDrawer&selectedStory=Notification%20Drawer&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs)

File: package.json [hunk 2]
```diff
@@ -13,15 +13,15 @@
     "cytoscape-klay": "3.1.1",
     "cytoscape-popper": "^1.0.1",
```
Comment (josejulio) [inline package.json]: While you are here, you might also *lock* the `cytoscape-popper` version, WDYT?

**Comment by mtho11** [comment]
The strange thing is I did commit my package-lock.json as you can see in the commit. This is weird.

File: package.json (hunk 1 above)
Comment (mtho11) [reply package.json]: Ok.

Comment (josejulio) [reply package.json]: Yep, actually that's the version we need for the NotificationDrawer, not the 1.16.6 :)

**Pull request merged**